app/utils/callback/lava.py

```diff
@@ -584,6 +589,11 @@ def add_tests(job_data, job_meta, lab_name, db_options,
     :type base_path: string
     :return The top-level test group document id as ObjectId object.
     """
+    if job_data.get("status") not in (COMPLETE,):
```

Comment:
Same thing here about log messages.

Also, I think we may actually not be able to do the `login` test case if we skip `INCOMPLETE` jobs, as if the login failed I believe the LAVA job status is going to be `INCOMPLETE`.  It's something worth verifying though.